aws/cloudformation-templates/services/service/service.yaml

```diff
@@ -266,6 +266,7 @@ Resources:
 
       ManagedPolicyArns:
         - arn:aws:iam::aws:policy/AWSCloudMapDiscoverInstanceAccess
+        - arn:aws:iam::aws:policy/AWSXrayFullAccess
```

Comment:
Is full access required or can this be scoped down?

Reply:
scoped down to  arn:aws:iam::aws:policy/AWSXRayDaemonWriteAccess 
It's the best we can do (X-ray doesn't support resource level on this action)